lib/game.rb

```diff
@@ -0,0 +1,142 @@
+class Game 
+
+  WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize (player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @board.turn_count.even? ? player_1 : player_2
+  end
+
+  def over?
+    won? || draw? || board.full? ? true : false
```

Comment:
The last term, `board.full? ? true : false`, is already covered by the `draw?` method. This could be reduced to:
```
def over?
  won? || draw?
end
```